resources/views/engagements/confirm-access-needs.blade.php

```diff
@@ -1,26 +1,34 @@
 <x-app-wide-layout>
-    <x-slot name="title">{{ __('Confirm access needs') }}</x-slot>
+    <x-slot name="title">{{ __('Confirm your access needs') }}</x-slot>
     <x-slot name="header">
         <ol class="breadcrumbs" role="list">
             <li><a href="{{ localized_route('projects.my-projects') }}">{{ __('My projects') }}</a></li>
             <li><a href="{{ localized_route('projects.show', $project) }}">{{ $project->name }}</a></li>
             <li><a href="{{ localized_route('engagements.show', $engagement) }}">{{ $engagement->name }}</a></li>
         </ol>
         <h1 class="w-full md:w-2/3">
-            {{ __('Confirm access needs') }}
+            {{ __('Confirm your access needs') }}
         </h1>
     </x-slot>
 
     <div class="stack mb-12 w-full md:w-2/3">
         <p>{{ __('You have successfully signed up, and your name and contact information have been shared with :projectable.', ['projectable' => $project->projectable->name]) }}
         </p>
 
-        <h2>{{ __('What’s been shared anonymously') }}</h2>
+        <h2>{{ __('What’s been shared') }}</h2>
 
         {!! Str::markdown(
             __(
-                'The following access needs have been shared with :projectable **anonymously**. They will **not** know it is you who requested them.',
-                ['projectable' => $project->projectable->name],
+                'Based on what you’ve selected in your :access_needs_settings_link, :projectable has been asked to provide the following access supports.',
+                [
+                    'access_needs_settings_link' =>
+                        '<a href="' .
+                        localized_route('settings.edit-access-needs') .
+                        '">' .
+                        __('access needs settings') .
+                        '</a>',
+                    'projectable' => $project->projectable->name,
+                ],
             ),
         ) !!}
```

Comment:
This is using `Str:markdown` but the string doesn't contain markdown; however there is html added. I think we can either remove `Str:markdown` or add the links with markdown syntax.

Reply:
Good call. It used to have Markdown. I think you are right that we can remove it and wrap in a paragraph.